src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+/**
+ * Represents a task to be done. This task possesses a state/status that is by default not done.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Creates a Task that is initially undone.
+     *
+     * @param description Description of Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
```suggestion
        // mark done task with X
        return (isDone ? "X" : " "); 
```
I'm not sure but maybe the comment is supposed to be like this?